Firestore/core/src/firebase/firestore/remote/remote_objc_bridge.mm

```diff
@@ -60,10 +60,15 @@
   return output.str();
 }
 
-NSData* ConvertToNsData(const grpc::ByteBuffer& buffer) {
+NSData* ConvertToNsData(const grpc::ByteBuffer& buffer, NSError** out_error) {
   std::vector<grpc::Slice> slices;
   grpc::Status status = buffer.Dump(&slices);
-  HARD_ASSERT(status.ok(), "Trying to convert an invalid grpc::ByteBuffer");
+  if (!status.ok()) {
```

Comment:
This may happen if the `buffer` was empty. I'm not sure if gRPC may return an empty buffer, but decided to err on the side of caution.